### DISCUSSION TITLE
Adapter wont update

### DISCUSSION THREAD

**Discussion by mazzylp94** [darthcloud/BlueRetro#982] [Q&A]
(solved) i conect the device on the front usb 3.0.... then conect to a usb 2.0 on the back and problem solved update finished

Hello. this is my first post and sorry for my bad english im from argentina

today i recive and adapter that i buy in aliexpress. the box says  first i have to update the bueretro core.

scan the QR and take me to www.blue-retro.com , then a link to this github

i download the update and the app on windows 10, load the bin files and then pres start but the update wont start, says SYNC  and a ...... loading bar but didnt update (the pc recognizes the device on com port 3)

here is a screnshot . thanks

![scren1](https://github.com/darthcloud/BlueRetro/assets/168477311/3d280ef7-7cc6-4bd3-9260-c3a63263b581)

**Comment by mazzylp94** [comment]
SOLVED just conect to a usb 2.0 port not 3.0

**Reply by treyus30** [reply]
This did not work for me either

**Reply by treyus30** [reply]
For anyone else, please see my solution:
https://github.com/darthcloud/BlueRetro/discussions/1025
